### PR TITLE
Remove deps

### PR DESCRIPTION
**Pull request by tynes**
- Switch away from `winston` to `blgr`
- Remove some deps from `package.json` that are no longer in use

### CODE REVIEW

File: server/logger.js [hunk 1]
```diff
@@ -9,31 +9,15 @@ if (!fs.existsSync(logDir)) {
...
-      json: false,
-      colorize: true
-    })
-  ],
-  exitOnError: false
+const logger = new blgr();
+logger.set({
+  level: env === 'development' ? 'debug' : 'info',
+  colors: true,
+  filename: logDir.concat('/logs.log')
```
Comment (bucko13) [inline server/logger.js]: Thoughts on this location for the logs (gets set to `__dirname/../logs` above)? This was before we were using bcfg and before there was a `prefix`. Would logs be better in `~/.bpanel` (or wherever a custom prefix is set to)?

File: server/logger.js [hunk 2]
```diff
@@ -9,31 +9,15 @@ if (!fs.existsSync(logDir)) {
...
-      level: 'debug',
-      handleExceptions: true,
-      json: false,
-      colorize: true
-    })
-  ],
-  exitOnError: false
+const logger = new blgr();
+logger.set({
+  level: env === 'development' ? 'debug' : 'info',
```
[the remark this answers is not in the document]
Comment (tynes) [reply server/logger.js]: It totally makes sense to have only 1 instance. Didn't realize there was another instance being instantiated, I tried to minimally remove `winston`

File: server/logger.js [hunk 3]
```diff
@@ -9,31 +9,15 @@ if (!fs.existsSync(logDir)) {
...
+  colors: true,
+  filename: logDir.concat('/logs.log')
 });
 
-/*
-  Stream is used to allow morgan to style console output
-  in server.js
-*/
+logger.open();
```
Comment (bucko13) [inline server/logger.js]: This is also an async operation. I wonder if this could cause some problems when enabling file logging.

**Comment by bucko13** [comment]
Testing this out locally, I think this also breaks server tests as there is some logger integration (specifically for `configHelpers`)

**Pull request closed without merging**